INSTRUCTION.md

```diff
@@ -0,0 +1,89 @@
+## Deploy the application following the commands:
+```
+cd ./.infrastructure/  
+kubectl apply -f namespace.yml
+kubectl apply -f busybox.yml
+kubectl apply -f clusterip.yml
+kubectl apply -f nodeport.yml
+kubectl apply -f deployment.yml
+kubectl apply -f hpa.yml
+kubectl apply -f todoapp-pod.yml
+cd ..
+```
+## Check Pod Status:
+```
+kubectl get pods -o wide -n todoapp
+```
+
+## Go to the pod busybox:
+```
+kubectl -n todoapp exec -it busybox --sh
```

Comment:
Similar to the previous comment, the namespace `todoapp` is used here, but it should be `mateapp` to match the deployment and HPA configurations.